servers/republik/lib/MembershipStats/evolution.js

```diff
@@ -219,7 +219,53 @@ const populate = async (context, resultFn) => {
   await createCache(context).set({ result, updatedAt: new Date() })
 }
 
+/**
+ * @typedef {Object} AddSubtractProps
+ * @property {string[]} [add] Props to add to sum
+ * @property {string[]} [subtract] Props to subtract from sum
+ */
+
+/**
+ * Sum bucket properties from cached data
+ *
+ * @param {string} [key] Bucket key e.g. "2020-07"
+ * @param {AddSubtractProps} [props] Add or subtract passed properties
+ *
+ * @returns {(number|null)}
+ */
+const sumBucketProps = async (
+  context,
+  key = moment().format('YYYY-MM'),
+  props
+) => {
+  // Fetch pre-populated data
+  const data = await createCache(context).get()
+
+  // Return null in case pre-populated data is not available
+  if (!data) {
+    return null
+  }
```

Comment:
```suggestion
  if (!data) {
    console.error('pre-populated data is not available')
    return null
  }
```

Reply:
Logging very good, using WARN level instead. Keeping ERROR level for those backends is throwing, too.